mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -202,8 +199,8 @@ where
     }
 }
 
-pub async fn retry_subscriptions(
-    retries: UnboundedReceiver<SubscribeTo>,
+pub async fn retry_subscriptions<S: Stream<Item = SubscribeTo> + Unpin>(
+    retries: S,
```

Comment:
Why did this one become generic?

Reply:
I hope at some point we get rid of UnboundedReceiverStream (one Stream is in stable rust)